dbms/src/Storages/KVStore/tests/gtest_async_tasks.cpp

```diff
@@ -28,34 +28,47 @@ TEST(AsyncTasksTest, AsyncTasksNormal)
     auto log = DB::Logger::get();
     LOG_INFO(log, "Cancel and addTask");
     // Cancel and addTask
-    // 3 -> 1 -> 4 -> 2
     {
         auto async_tasks = std::make_unique<TestAsyncTasks>(1, 1, 2);
         auto m = std::make_shared<std::mutex>();
         int flag = 0;
         std::unique_lock cl(*m);
         std::atomic_bool finished_flag = false;
-        async_tasks->addTask(1, [m, &flag, &async_tasks, &finished_flag]() {
+        std::atomic_bool running_flag = false;
+        async_tasks->addTask(1, [m, &flag, &async_tasks, &finished_flag, &running_flag]() {
```

Comment:
enclose L38-L63 into `{}` and L64-L75

Reply:
so do not need to rename `count` to `count2`

Reply:
fixed